metals/src/main/scala/scala/meta/internal/metals/codeactions/ImportMissingSymbol.scala

```diff
@@ -241,6 +241,11 @@ class ImportMissingSymbolQuickFix(
   override protected val allSymbolsTitle: String =
     ImportMissingSymbol.allSymbolsTitle
   override protected def isImportAllSourceAction: Boolean = false
+  override protected def filterImportActions(
+      allActions: Seq[l.CodeAction]
+  ): Seq[l.CodeAction] = {
+    allActions.filter(!_.getTitle().endsWith("from package '<empty>'"))
```

Comment:
This should be fixed in AutoImportsProvider, there should be an option to just check if package is empty there and filter it this way